### DISCUSSION TITLE
Possible to update multiple A records?

### DISCUSSION THREAD

**Discussion by MrSupaBeast** [fire1ce/cloudflareDDNS-PowerShell#4] [Q&A]
Is it possible to update multiple A records (@ and the www subdomain) and how is this done if possible?

**Comment by fire1ce** [comment]
@MrSupaBeast, the short answer is no. It's a DDNS script. not an api script for cloudflare.

**Comment by AriZoNaiCe** [comment]
Theoretically, it can support updating multiple records to the same IP address.  You'd need to update/edit the existing script to add a foreach and maybe act on a comma-separated-list.  Should not be a very heavy lift.  The other option, of course, is to create x number of copies of the script, one for each record you want updated.  Then you'd need an equal number of Task Scheduler jobs as well.  Less clean, in my opinion.

**Comment by AriZoNaiCe** [comment]
[update-cloudflare-dns_multiple_records.zip](https://github.com/user-attachments/files/20747776/update-cloudflare-dns_multiple_records.zip)

I went ahead and completed this for me and thought I'd share, in case anyone else wants it.  @fire1ce - feel free to adopt this if you find it useful.  I did not look to handle entries in multiple zones or proxied records.  The only major adjustment I made was renaming the $dns_record variable in the variables file to $dns_record_list to support the comma-separated list.

Hope that helps someone.
